test-end-to-end/android-maestro/common/add_existing_content_block_to_course.yaml

```diff
@@ -0,0 +1,13 @@
+appId: com.toughra.ustadmobile
+---
+#this flow add content that is already added to library by admin as a content block inside course
+
+# edit the course to add blocks
+- tapOn: "Edit"
+- swipe:
```

Comment:
Why the swipe here (scroll down)? Please add a comment on that.

Reply:
Add block button will be only visible after scrolling to down, I have added a comment. Thanks